plugins/postcss-cascade-layers/src/constants.ts

```diff
@@ -9,6 +9,7 @@ export const IMPLICIT_LAYER_SUFFIX = 'b147acf6-11a6-4338-a4d0-80aef4cd1a2f';
 export const CONDITIONAL_ATRULES = [
 	'media',
 	'supports',
+	'container',
```

Comment:
With `@container` shipping in browsers we need to make sure we handle it correctly.